module-system/module-implementations/sov-attester-incentives/src/query.rs

```diff
@@ -44,22 +44,25 @@ where
         }
     }
 
+    /// Gives storage key for given address
+    pub fn get_attester_storage_key(&self, address: C::Address) -> StorageKey {
+        let prefix = self.bonded_attesters.prefix();
+        let codec = self.bonded_attesters.codec();
+        StorageKey::new(prefix, &address, codec)
+    }
+
     /// Used by attesters to get a proof that they were bonded before starting to produce attestations.
     /// A bonding proof is valid for `max_finality_period` blocks, the attester can only produce transition
     /// attestations for this specific amount of time.
+    #[cfg(feature = "native")]
     pub fn get_bond_proof(
         &self,
         address: C::Address,
-        witness: &<<C as Spec>::Storage as Storage>::Witness,
         working_set: &mut WorkingSet<C>,
-    ) -> StorageProof<<C::Storage as Storage>::Proof>
-    where
-        C::Storage: NativeStorage,
```

Comment:
Isn't  ` C::Storage: NativeStorage` needed to avoid: https://github.com/Sovereign-Labs/sovereign-sdk/pull/978/files#r1360613601